* * * * * /usr/bin/python3 /home/s/d_db.py 2> err.out

